src/main/resources/org/terasology/launcher/packages/schema.json

```diff
@@ -13,17 +13,8 @@
         "type": "array",
         "items": {
           "oneOf": [
-
-            {"type": "string", "minLength": 1},
-
-            {
-              "type": "object",
-              "properties": {
-                "id": {"type": "string", "minLength": 1},
-                "name": {"type": "string", "minLength": 1}
-              },
-              "required": ["id", "name"]
-            }
+            {"$ref": "subschema/TrackedPackage.json#/4.0"},
+            {"$ref": "subschema/TrackedPackage.json#/4.1"}
```

Comment:
- I'm not sure about the versioning here - both JSON schemas are actually for version 4.0 (and this is more or less an exercise how to deal with it in future versions). I would rather got with simple `v1`, `v2`, ... 
- With this approach, can we figure out to which JSON schema a file adheres? This will be important when trying to migrate schemas between versions.
- This differentiates only the `trackedPackages` field, not the whole schema itself... (belongs to the bullet point above)

Reply:
:+1:  for using v1, v2, ...
Since I'm essentially breaking down the schema into multiple subschemas, each of which could potentially have their own versions, it'd look more bloated than using multiple versions of the top schema (`schema-v1.json`, `schema-v2.json`, ...). Right now, I'm not sure of the advantages of one over the other.